src/server/command_registry.h

```diff
@@ -160,11 +188,13 @@ class CommandId : public facade::CommandId {
   }
 
  private:
+  // The following fields must copy manually in the move constructor.
   bool implicit_acl_;
+  bool is_alias_{false};
   std::unique_ptr<CmdCallStats[]> command_stats_;
   Handler3 handler_;
   ArgValidator validator_;
-  bool is_alias_{false};
+  MoveOnly<struct hdr_histogram*> latency_histogram_;  // Histogram for command latency in usec
```

Comment:
Nit: the `struct` type specifier is redundant here and elsewhere with `hdr_histogram`